wren.go

```diff
@@ -318,6 +344,51 @@ func write(vm *C.WrenVM, text *C.char) {
 	fmt.Fprint(out, C.GoString(text))
 }
 
+//helper
+func readModule(dir string, name string) (string, error) {
+	var err error
+
+	// Precedence (dir/name.wren) next (dir/name/module.wren)
+	filename := filepath.Join(dir, name+".wren")
+	if _, err = os.Stat(filename); os.IsNotExist(err) {
+		filename = filepath.Join(dir, name, "module.wren")
+		if _, err = os.Stat(filename); os.IsNotExist(err) {
+			return "", err
+		}
+	}
+	var data []byte
+	if data, err = ioutil.ReadFile(filename); err != nil {
+		return "", err
+	}
+	return string(data), nil
+}
+
+//export loadModule
+func loadModule(vm *C.WrenVM, name *C.char) *C.char {
+	var module string = C.GoString(name)
+	var source string
+
+	// Ensure module does not have undesired characters
+	// that can pose thread to remote-code-inclusions
+	if !strings.Contains(module, "..") {
+		// Proceed to load from the configured modules directory only
+		var jvalPtr unsafe.Pointer = C.wrenGetUserData(vm)
+		if jvalPtr != nil {
+			userData := make(map[string]interface{})
+			jval := C.GoString((*C.char)(jvalPtr))
+			if e := json.Unmarshal([]byte(jval), &userData); e == nil {
```

Comment:
It looks like this is not doing anything with the error if it's non-nil. Since I assume this method can't have a return type of `(*C.char, error)`, is there any other way to let Wren know about the error? At the very least, problems encountered here should be logged.

Reply:
Empty source indicates desired module unavailability and eventual makes Wren signal error. So this should not be a trouble.